### PR TITLE
Adds possible fix to latex builds

### PR DESCRIPTION
**Pull request by dpshelio**

### CODE REVIEW

File: latex.tplx [hunk 1]
```diff
@@ -1,3 +1,28 @@
+%=====================================
+% Solution about utf8x problem
+% https://github.com/jupyter/nbconvert/issues/530#issuecomment-303034557
+%=====================================
+
+((*- extends 'article.tplx' -*))
```
Comment (ageorgou) [inline latex.tplx]: I think this is why the build fails on Travis: A template can extend at most one other template [(Jinja docs)](http://jinja.pocoo.org/docs/2.10/templates/#extends), and we are already extending `ipython_style.tplx` later on. We can probably just move all this after the existing `((* extends cell_style *))` instruction. That template also has a `packages` block, so it should work.

File: latex.tplx [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+[unrelated template changes]
```
Comment (ageorgou) [inline latex.tplx]: I think this line was just an example in the suggested workaround, so we don't need it!

File: Makefile [hunk 3]
```diff
@@ -28,8 +29,11 @@ default: _site
 %.png: %.dot Makefile
 	dot $< -T png -o $@
 
-%.png: %.uml Makefile
-   java -Djava.awt.headless=true -jar plantuml.jar -p < $< > $@
+%.png: %.svg Makefile
```
Comment (giordano) [inline Makefile]: Why is `Makefile` a prerequisite? :thinking:

**Comment by ageorgou** [comment]
Apparently `latex-xcolor` doesn't exist in bionic. However, [in Xenial it was included in](https://packages.ubuntu.com/xenial/latex-xcolor) `texlive-latex-recommended`.

**Comment by ageorgou** [comment]
I'm not sure what's happened to `texlive-math-extra`, which doesn't exist in Bionic either. [This thread](https://tex.stackexchange.com/questions/158536/texlive-math-extra-and-texlive-latex-extra-packages-cannot-be-installed) suggests using `texlive-science`, although that's a different package that also existed in Xenial (and the thread is supposed to be about Fedora, anyway).

File: .travis.yml [hunk 4]
```diff
@@ -17,9 +17,9 @@ addons:
     - texlive-latex-recommended
     - texlive-generic-recommended
     - texlive-generic-extra
-    - texlive-math-extra
+    - texlive-science
     - texlive-xetex
-    - latex-xcolor
+    - texlive-latex-recommended
```
Comment (ageorgou) [inline .travis.yml]: This is already included above :)

**Comment by ageorgou** [comment]
This looks good, except I don't fully understand the changes around the svg/png files. As the PR Travis build fails because of a missing png, I won't merge it just yet, in case there is something there that causes the error!

**Pull request merged**